airbyte-integrations/connectors/destination-langchain/metadata.yaml

```diff
@@ -0,0 +1,17 @@
+data:
+  registries:
+    oss:
+      enabled: false
+  connectorSubtype: database
+  connectorType: destination
+  definitionId: cf98d52c-ba5a-4dfd-8ada-c1baebfa6e73
+  dockerImageTag: 0.1.0
+  dockerRepository: airbyte/destination-langchain
+  githubIssueLabel: destination-langchain
+  icon: langchain.svg
+  license: MIT
+  name: Langchain
```

Comment:
does it make more sense to call this something like `Vector Database (powered by LangChain)`?